hnn_core/network.py

```diff
@@ -1665,15 +1672,33 @@ def _add_cell_type_bias(network: Network, amplitude: Union[float, dict],
                              f'{list(network.cell_types.keys())}. '
                              f'Got {cell_type}')
 
-        if 'tonic' not in network.external_biases:
-            network.external_biases['tonic'] = dict()
+        if bias_name not in network.external_biases:
+            network.external_biases[bias_name] = dict()
 
-        if cell_type in network.external_biases['tonic']:
-            raise ValueError(f'Tonic bias already defined for {cell_type}')
+        if cell_type in network.external_biases[bias_name]:
+            raise ValueError(f'Bias named {bias_name} already defined '
+                             f'for {cell_type}')
 
         cell_type_bias = {
             'amplitude': amplitude,
             't0': t_0,
             'tstop': t_stop
         }
-        network.external_biases['tonic'][cell_type] = cell_type_bias
+
+        sections = list(network.cell_types[cell_type].sections.keys())
+
+        # Ensures backwards compatibility with existing json
+        # files (see test_read_configuration_json in test_io.py)
+        if section is None:
+            warnings.warn("Section for tonic bias is not defined."
+                          "Defaulting to 'soma'.",
+                          DeprecationWarning, stacklevel=1)
+        # error when section is defined that doesn't exist.
+        elif section not in sections:
+            raise ValueError(f'section must be an existing '
+                             f'section of the current cell or None. '
+                             f'Got {section}.')
+        else:
+            cell_type_bias['section'] = section
```

Comment:
Where is it defaulting to soma?

Reply:
I realize now that it just doesn't add 'section' to the cell_type_bias dictionary and the `Cell.create_tonic_bias` will eventually use 'soma' as a default. I feel like that behavior is obtuse. Is it possible to bring the defaults higher-level?

Reply:
See above, it's because of test_read_configuration_json

Reply:
Resolved in developer meeting. We ended up regenerating the jones2009_3x3_drives.json file so that it contains section. The user will not be affected by this change if they don't define a section.